src/fsae_drivers/fsae_pins.c

```diff
@@ -0,0 +1,109 @@
+#include "fsae_pins.h"
+
+#include "chip.h"
+
+// Low Side Contactor
+#define FSAE_CTR_LOW_SWTCH 2, 8
+#define FSAE_IOCON_CTR_LOW_SWTCH IOCON_PIO2_8
+
+// High Side Contactor
+#define FSAE_CTR_HIGH_SWTCH 2, 7
+#define FSAE_IOCON_CTR_HIGH_SWTCH IOCON_PIO2_7
+
+// Fault Pin
+#define FSAE_FAULT_GPIO 3, 0
+#define FSAE_IOCON_FAULT_GPIO IOCON_PIO3_0
+
+// Charge Enable Pin
+#define FSAE_CHARGE_ENABLE_GPIO 3, 2
+#define FSAE_IOCON_CHARGE_ENABLE_GPIO IOCON_PIO3_2
+
+// Fan 1 Pin
+#define FSAE_FAN_1_PIN 1, 4
+#define FSAE_IOCON_FAN_1_PIN IOCON_PIO1_4
+
+// Fan 2 Pin
+#define FSAE_FAN_2_PIN 1, 3
+#define FSAE_IOCON_FAN_2_PIN IOCON_PIO1_3
+
+#define FAN_TIMER_PRESCALE 18
+#define MATCH_REGISTER_FAN_1 3
+#define MATCH_REGISTER_FAN_2 2
+#define ENABLE_PWM_FAN_1 0b1000
+#define ENABLE_PWM_FAN_2 0b0100
+#define MATCH_REGISTER_FAN_PWM_CYCLE 0
+#define FAN_OFF_DUTY_RATIO -100
+#define FAN_ON_DUTY_RATIO 10
+#define FAN_PWM_CYCLE 100
+
+void Fsae_GPIO_Init(void) {
+    // LEDs
+    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED1);
+    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED2);
```

Comment:
There are 2 steps to initialize a GPIO
1. Set the GPIOnDIR register using Chip_GPIO_SetPinDIROutput() or Chip_GPIO_WriteDirBit() (I prefer Chip_GPIO_SetPinDIROutput() because it's easier to understand)
2. Set the IOCON register using Chip_IOCON_PinMuxSet()
3. (if you want) Initialize the state of the GPIO using Chip_GPIO_SetPinState()

Page 106 of the user manual talks about IOCON and GPIOnDIR registers